CDCgov/prime-reportstream
# 11672: NDL mappings

This PR adds the mapping of NDL -> PractitionerRole and CNN -> Practitioner

Test Steps:
1. Make sure the integration tests pass
2. Verify that the new mappings follow
  - https://docs.google.com/spreadsheets/d/11OXFdsd0KLUaUQRmYawlszF5dcRZQ6CGJudPaBFTmPM/edit#gid=0
  - https://docs.google.com/spreadsheets/d/10RIiHor6MGnmIBrj1tKkhW2kkrgvODeTjDpJAj3R40Q/edit#gid=0

## Changes
- Mappings for CNN -> Practitioner -> CNN
- Mappings for NDL -> PractitionerRole -> NDL

## Checklist

### Testing

### Process

## Linked Issues
- #11672 

## To Be Done
- Ticket for updating hl7v2 library to support `partOf` use case

## Specific Security-related subjects a reviewer should pay specific attention to
